xml/compute.xml

```diff
@@ -870,9 +870,9 @@
   </sect2>
  </sect1>
  <sect1 xml:id="FileFormat">
-  <title>File Format Libraries</title>
+  <title>File format libraries</title>
   <sect2 xml:id="sec2-lib-adios">
-   <title>Adaptable IO System (<literal>ADIOS</literal>) 1.13.1 Has Been Added</title>
+   <title>Adaptable IO system (<literal>ADIOS</literal>) 1.13.1 has been added</title>
```

Comment:
Again, since it's an acronym, I'd leave this.

Reply:
```suggestion
    <title>Adaptable IO System (ADIOS)</title> 
```